pkg/handler/eventhandler.go

```diff
@@ -136,7 +136,7 @@ func (h TypedFuncs[object, request]) Create(ctx context.Context, e event.TypedCr
 					priority = LowPriority
 				}
 				q.(priorityqueue.PriorityQueue[request]).AddWithOpts(
-					priorityqueue.AddOpts{Priority: priority},
+					priorityqueue.AddOpts{Priority: &priority},
```

Comment:
Should this only be set if IsInInitialList is true?

(currently we otherwise set it to &0)

(same below)